client/app/scripts/liveblog-edit/views/main.html

```diff
@@ -127,8 +132,10 @@ <h3 class="page-nav-title">{{ blog.title }}</h3>
                       lb-posts-list
                       lb-posts-blog-id="blog._id"
                       lb-posts-status="open"
-                      lb-posts-allow-unpublish="true"
-                      lb-posts-allow-reordering="true"
+                      lb-posts-allow-unpublishing="blogSecurityService.canPublishAPost"
```

Comment:
We need this mapping here... This is the aim of Angular to do the mapping in the DOM...
and all the method start with *allow-*...